apps/meteor/server/models/raw/BaseRaw.ts

```diff
@@ -231,6 +231,10 @@ export abstract class BaseRaw<
 		};
 	}
 
+	exists(query: Filter<T>): Promise<boolean> {
+		return this.col.find(query, { limit: 1, batchSize: 1, projection: { _id: 1 } }).hasNext();
```

Comment:
just curious, what is the motivation behind this instead of using a `findOne`?

Reply:
Perf maybe? Not sure. At least hasNext dont return any data so it should be a bit nicer on network. But since we're removing the only usage of it i'll remove this as well.